_episodes/02-enter-sandman.md

```diff
@@ -162,7 +161,7 @@ else:
 ~~~
 {: .language-python}
 
-and then make it executable `chmod +x python_exit.py`. Now, try running it with `./python_exit.py hello` and `./python_exit.py goodbye` and see what those exit codes are. Déjà vu?
+and then make it an executable `chmod +x python_exit.py`.  Let's check the exit code with `echo $?`. Déjà vu?
```

Comment:
I'd leave the statement to run it with `./python_exit.py` here. Otherwise you jump from `chmod` to checking the exit code of `chmod`...

Reply:
Same above with `bash_exit.sh`.